dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -358,6 +358,26 @@ const mostPopContainerStyles = css`
 	}
 `;
 
+const liveBlogTopAdStyles = css`
+	min-height: ${adSizes.mpu.height + labelHeight}px;
+	min-width: ${adSizes.mpu.width}px;
+	width: fit-content;
+	max-width: ${adSizes.mpu.width}px;
```

Comment:
Is this different from just setting `width`?

```suggestion
	width: ${adSizes.mpu.width}px;
```

Reply:
I've copied this from other slots, I believe this nonsense might be because Google sometimes adds inline `width` styles

All this ad CSS needs tidying at some point!